### ISSUE TITLE
The Angular app host has a wrong NavBar UX

### ISSUE THREAD

**Issue by kdinev** [IgniteUI/igniteui-cli#310]
### Description  
The `NavBar` title in the `AppHost` is hardcoded but is not the app name, which according to material, the title should be related to the screen you've navigated to, or should be the application name.

https://material.io/design/components/app-bars-top.html#anatomy

- Ignite UI CLI version: 1.3.3
- Framework: Angular
- Project type: Ignite UI for Angular

```html
  <div igxFlex class="main">
    <igx-navbar title="Ignite UI for Angular" actionButtonIcon="menu" (onAction)="nav.toggle()">
    </igx-navbar>
    <div class="content">
      <router-outlet></router-outlet>
     </div>
  </div>
```

### Steps to reproduce  

1. Create a new project
2. Add a component
3. Run and navigate to the component

### Result  
The `NavBar` title stays the same, but is not the app name.

### Expected result  
The `NavBar` title should change to the new route name, or it should be set to the application name.

**Comment by IvayloG** [comment]
1. This is fixed some time ago in [v1.3.0-beta.1](https://github.com/IgniteUI/igniteui-cli/blob/v1.3.0-beta.1/templates/angular/igx-ts/projects/empty/files/src/app/app.component.html). 
The result is: The NavBar title is set to the application name and persisted during navigation.  

2. I have tested this locally with Ignite UI CLI version: 1.3.3 and works as expected.
![image](https://user-images.githubusercontent.com/4296944/42989115-8a963d2e-8c07-11e8-9477-cee53421e1bb.png)

3. Let me know if there are some other steps/specifics to reproduce the issue.

**Comment by IvayloG** [comment]
I am closing this as no longer reproducible.
